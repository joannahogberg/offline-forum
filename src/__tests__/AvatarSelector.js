import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import AvatarSelector from '../components/AvatarSelector';

describe('<AvatarSelector />', () => {
  it('should display the zac avatar', () => {
    const wrapper = shallow(<AvatarSelector currentPersona='Zac' />);
    expect(wrapper.find('img').prop('src')).toEqual('zac.png');
  });

  it('should display the morgana avatar', () => {
    const wrapper = shallow(<AvatarSelector currentPersona='Morgana' />);
    expect(wrapper.find('img').prop('src')).toEqual('morgana.png');
  });

  it('should display the esmeralda avatar', () => {
    const wrapper = shallow(<AvatarSelector currentPersona='Esmeralda' />);
    expect(wrapper.find('img').prop('src')).toEqual('esmeralda.png');
  });
});

// Snapshot
it('should display the zac avatar', () => {
  const tree = renderer
    .create(<AvatarSelector currentPersona='zac' />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
