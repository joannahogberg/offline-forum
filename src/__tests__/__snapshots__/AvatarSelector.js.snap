// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display the zac avatar 1`] = `
<img
  alt="Avatar"
  className="w-12 rounded-full"
  src="zac.png"
  style={
    Object {
      "transform": "translateY(20px)",
    }
  }
/>
`;
